packages/x-components/tests/e2e/extra-params.feature

```diff
@@ -1,33 +1,41 @@
 Feature: Extra-params component
 
   Background:
-    Given a next queries API
-    And   a results API
-    And   a query suggestions API
-    And   a popular searches API
-    And   a related tags API
+    Given a next queries API with a known response
+    And   a results API with a known response
+    And   a query suggestions API with a known response
+    And   a popular searches API with a known response
+    And   a related tags API with a known response
     And   a recommendations API with a known response
+    And   a tracking API
     And   no special config for layout view
     And   start button is clicked
 
-  Scenario Outline: 1. Search request includes extra-params from Snippet Config
-    Given a results API with a known response
+  Scenario Outline: 1. Search request includes extra-params
```

Comment:
Both tests were testing the same initially, so I decided to join them and extend them to test everything we need.